### DISCUSSION TITLE
Pallene still be actively developed/used?

### DISCUSSION THREAD

**Discussion by tcoram** [pallene-lang/pallene#592] [General]
Hi, 
I only recently stumbled upon Pallene, but have noticed that most presentations/discussions/etc, I've found,  are from a year or more ago.
The last git commit here is several months ago.

I've been considering using Pallene for an embedded project (where type checks and performance would be useful)... is that a good idea at this point?

/todd

**Comment by hugomg** [comment]
I wasn't able to dedicate much time to Pallene in the last couple of years, because I was was working on non-Lua research. But this year I'm starting a new job where I can dedicate more time to Pallene. I want to get it back on track this year. By the way, help is always welcome :)

**Reply by bjornbm** [reply]
Glad to hear this news!

**Reply by tcoram** [reply]
Thanks for the prompt reply!  That's good news.  If I am able to use it on my project, then I will definitely be able to actively contribute (e.g. experiences using it, bug reports, examples, suggestions, etc).  I just wanted to make sure that  Pallene wasn't abandoned.
